### DISCUSSION TITLE
What is it about `ui.buffer_statusbar_text`?

### DISCUSSION THREAD

**Discussion by metaleap** [orbitalquark/textadept#150] [Q&A] [answered]
I mean all is clear on `ui.statusbar_text`, it renders what it's set to, and its value reflects what the statusbar shows.

But `ui.buffer_statusbar_text`? _Setting it_ I don't see that text anywhere, and somehow it's write-only. Presumably it's about the right-hand-side buffer-specific items, but why is it part of the "scripter API" when it doesn't show anywhere? Or just some WIP thing?

**Comment by rgieseke** [comment, marked as answer]
You can see it in action here:
https://github.com/orbitalquark/textadept/blob/3ce1997b68b0454840870e35d84a4bb5feddfab3/core/ui.lua#L330

If you want to run your own, you need to create an event which runs after the original update_ui event handler, because the original event would run after you just write something with `ui.buffer_statusbar_text` from the command entry.

```lua
events.connect(events.UPDATE_UI, function()
  ui.buffer_statusbar_text = "Foo"
end)
```

**Reply by metaleap** [reply]
Nice to know, danke @rgieseke!
